benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/DataGenerator.java

```diff
@@ -15,6 +15,10 @@
 
```

Comment:
Looks good, only thing I am thinking is that the exceptions don't get propagated so the use of the DataGenerator doesn't know there has been a failure. We should probably just let them flow through the the caller to handle so it can act accordingly.

Reply:
done